cynthion/python/src/commands/util.py

```diff
@@ -41,14 +46,21 @@ def _find_assets_path():
     return os.path.normpath(os.path.join(pkg_path, assets))
 
 
+def _is_source_install():
+    return os.path.basename(files("cynthion")) == "src"
```

Comment:
This doesn't work for me. When I install from source, `os.path.basename(files("cynthion"))` is `'cynthion'`. Maybe we should just always include `_MSG_SOURCE_INSTALL`?

Reply:
Ah yes, I failed to take into account that not everyone is going to `pip install -e` 🤦

I've added a better check.

Reply:
https://github.com/greatscottgadgets/cynthion/pull/166/commits/eeec9ae47a0eef628a65fe59df37ef5107da7adc